core/input/input_event.cpp

```diff
@@ -1186,14 +1186,14 @@ static const char *_joy_button_descriptions[(size_t)JoyButton::SDL_MAX] = {
 };
 
 String InputEventJoypadButton::as_text() const {
-	String text = "Joypad Button " + itos((int64_t)button_index);
+	String text = vformat(RTR("Joypad Button %d"), (int64_t)button_index);
 
 	if (button_index > JoyButton::INVALID && button_index < JoyButton::SDL_MAX) {
-		text += vformat(" (%s)", _joy_button_descriptions[(size_t)button_index]);
+		text += vformat(" (%s)", TTRGET(_joy_button_descriptions[(size_t)button_index]));
 	}
 
 	if (pressure != 0) {
-		text += ", Pressure:" + String(Variant(pressure));
+		text += ", " + RTR("Pressure: ") + String(Variant(pressure));
```

Comment:
As mentioned in that other issue, trailing space should never be in the translated string. This should be:
```suggestion
		text += ", " + RTR("Pressure:") + " " + String(Variant(pressure));
```

Reply:
I forgot that's an option ;) Thought we have to ignore the warning anyway.